### ISSUE TITLE
Provide a mechanism to make fetch calls to sites that are using "weak" ciphers

### ISSUE THREAD

**Issue by ejose19** [denoland/deno#11407]
I've stumbled over many sites that are using "weak" ciphers, and deno doesn't provide a mechanism to make fetch calls to those sites, it instead errors out with:

```
TLS alert received: Message {
    typ: Alert,
    version: TLSv1_2,
    payload: Alert(
        AlertMessagePayload {
            level: Fatal,
            description: HandshakeFailure,
        },
    ),
}
```

I've checked #10447 and while it's related, this is more of a broad issue rather than a single site, because contacting every affected website operator and get them to upgrade their suite is not feasible. Instead, having an option to disable cipher alerts and proceed with the connection would be ideal, especially for those that are coming from node (where this limitation doesn't exists) and want to migrate/create an API based on deno.

**Comment by kitsonk** [comment]
It is effectively a duplicate of #10477. We don't want to support weak ciphers. No matter how alluring that is, weakening security for usability is not something we would consider lightly and in this case it is more than just a flag to allow it, the underlying libraries we use do not support weak and ciphers because they too don't want to open security vulnerabilities.

**Comment by kitsonk** [comment]
We also closed #6427 for the same reasons.
